src/libs/actions/User.js

```diff
@@ -237,7 +230,7 @@ function validateLogin(accountID, validateCode) {
         }
     }).finally(() => {
         Onyx.merge(ONYXKEYS.ACCOUNT, {isLoading: false});
-        Navigation.navigate(redirectRoute);
+        Navigation.navigate(ROUTES.HOME);
```

Comment:
I decided to just change the behavior here as I think it makes more sense this way. If you navigate to the `HOME` route it will open the drawer (which is where you would have been before validating a login via the settings page - which I presume is the `isLoggedIn` version of this).